components/feature/prompts/src/main/AndroidManifest.xml

```diff
@@ -5,7 +5,7 @@
     package="mozilla.components.feature.prompts">
     <application>
         <provider
-            android:authorities="mozilla.components.feature.prompts.fileprovider"
+            android:authorities="${applicationId}.fileprovider"
```

Comment:
Let's get this merged to fix the immediate problem for the app teams. However I wonder if we should still have some "android component" uniqueness in the name instead of just `fileprovider` which may be used by an app too (causing a conflict).